packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## v2.4.0-dev (Jan 22, 2024)
+- Fix invalid query params warnings [#1655](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1655)
```

Comment:
If you agree that these fixes are bug fixes, lets prefix this comment with BUGFIX so its easier for those doing releases to identify if they are making a feature release or a patch release.

Reply:
Looking for your thoughts on this comments still. Thanks

Reply:
I think it is considered bug fixes since we did passing invalid params in the template to commerce-react and then to the isomorphic. I've put the PR under bug fixes section. 2.4.0 is the next minor release. so this bug fix would be under the next patch release 2.3.2.

Should I add the patch version before the ### Bug fixes? 🤔